config/open-feature/org.yaml

```diff
@@ -198,9 +198,6 @@ repos:
     description: .NET implementation of the OpenFeature SDK
   dotnet-sdk-contrib:
     description: OpenFeature Providers and Hooks for .NET
-  feature-operator:
-    archived: true
-    description: OpenFeature Operator for Kubernetes
```

Comment:
This was a very old PoC version of the `open-feature-operator`, which was archived and collecting vulns in Snyk